lib/gradient/elixir_type.ex

```diff
@@ -0,0 +1,135 @@
+defmodule Gradient.ElixirType do
+  @moduledoc """
+  Module to format types.
+
+  TODO records
+  FIXME add tests
```

Comment:
Are these TODO/FIXME up to date? Can we create relevant issues in this repo for them?

Reply:
Yea, I don't provide any tests yet. For now, I only initialize the example file that can be used to produce some error messages. Records are not supported yet. Maybe I can try to finish these two things in this PR

Reply:
Let's keep PRs small, so they're easier to grok. We can do these other things in a separate PR / a few separate PRs. I was just making sure we keep track of the TODOs items in one place without need to grep code.